web/pages/api/og/market.tsx

```diff
@@ -36,16 +44,38 @@ const figtreeMediumData = fetch(FIGTREE_MED_URL).then((res) =>
   res.arrayBuffer()
 )
 
+async function getFreshOgMarketProps(
+  contractId: string | undefined
+): Promise<OgCardProps | undefined> {
+  if (contractId) {
+    try {
+      const contract = await getContract(db, contractId)
```

Comment:
It think it would be simpler to just use the [public api](https://docs.manifold.markets/api#get-v0marketmarketid) to fetch the contract rather than use the internal api. That way we don't have to patch lodash, either.

Reply:
I wasn't aware of the public API, thanks for the pointer.